### DISCUSSION TITLE
Scanning Qr Code

### DISCUSSION THREAD

**Discussion by ltbongo** [rainbow-me/rainbowkit#1946] [Q&A]
Trying to connect my Trust wallet from the phone to my site or even your site but neither the walletconnect QR or the QR from Trust Wallet is connecting me... 

Ive seen similar issues from users in the trustwallet support

**Comment by 0xmkh** [comment]
@ltbongo Sometimes WalletConnect can be unstable depending on what dApp you're using. Are you seeing this issue within your own project or somewhere else ?

**Comment by ltbongo** [comment]
The same issue also on the rainbowkit site… I cannot connect my mobile wallet to the site

**Reply by 0xmkh** [reply]
I have trust wallet and it connects fine for me. May i ask if you're using a vpn ?
